PMurphy1940/HandPickt
# Registration page setup

# Description
The registration and login page is functioning and redirecting to 'Dashboard' upon a successful result

## Type of change
Please delete options that are not relevant.

# Testing Instructions for REGISTRATION
The user may register a new account from the '/register' route. Account name and email address must be unique to register. Pre-existing values will reject. Password fields must match. Unique warnings will be displayed if any of the following occur:

  1) Account name is left blank
  2) Email is left blank
  3) Password and confirm password do not match
  4) Account name is already in use
  5) Email is already in use

Button to Show/Hide password entry
Link to Login page

If all fields are valid,
Register button will save the Account to the database, passwords are stored separately from the user object.
The User object is then acquired from the database with the assigned user id.
User is then placed into session storage (sans password) and directed to '/dashboard'

# Testing Instructions for LOGIN
The user may login from the '/login' route. Account name and password must match what is stored in the database. Unique warnings will be displayed if any of the following occur:

  1) Account name is left blank
  2) Account name is not found
  3) Password  is left blank
  4) Password  does not match stored value

Button to Show/Hide password entry
Link to Registration page

If all fields are valid,
Login button will place user into session storage (sans password) and user is directed to '/dashboard'

# Checklist:
